src/main/scala/com/cloudera/sparkts/ARIMA.scala

```diff
@@ -0,0 +1,566 @@
+/**
+ * Copyright (c) 2015, Cloudera, Inc. All Rights Reserved.
+ *
+ * Cloudera, Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"). You may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for
+ * the specific language governing permissions and limitations under the
+ * License.
+ */
+
+package com.cloudera.sparkts
+
+import breeze.linalg._
+
+import org.apache.commons.math3.analysis.{MultivariateVectorFunction, MultivariateFunction}
+import org.apache.commons.math3.optim.nonlinear.scalar.gradient.NonLinearConjugateGradientOptimizer
+import org.apache.commons.math3.optim._
+import org.apache.commons.math3.optim.nonlinear.scalar.{ObjectiveFunctionGradient, GoalType, ObjectiveFunction}
+import org.apache.commons.math3.optim.nonlinear.scalar.noderiv.BOBYQAOptimizer
+import org.apache.commons.math3.random.RandomGenerator
+import org.apache.commons.math3.stat.regression.OLSMultipleLinearRegression
+
+
```

Comment:
Add a short header comment explaining what ARIMA is and linking to the Wikipedia article